packages/debian/entrypoint.sh

```diff
@@ -38,15 +38,15 @@ rpm2deb() {
 
 buildrepo() {
     set -x
-    mkdir -p /repository/debian/{conf,incoming}
-    # All downloaded packages must be mounted in /metalk8s-deb
-    cp /metalk8s-deb/distributions /repository/debian/conf
-    #TODO: Mount `distributions` file in  /repository/debian/conf
-    reprepro -b /repository/debian export
-    for dir in /metalk8s-deb/metalk8s-*; do
-        reprepro -C "${dir##*/}" -b /repository/debian \
-            includedeb bionic "$dir"/*.deb
-    done
+
+    local -r reponame="$1"
+    local -r repodir="/metalk8s-deb/$1"
+
+    mkdir -p /repository/{conf,incoming}
+    cp /metalk8s-deb/distributions /repository/conf
```

Comment:
`cp /distributions /repository/conf`

That way we can mount `/metalk8s-deb` (or `/packages` if you follow my previous comment) as read-only.